base_layer/contacts/src/contacts_service/handle.rs

```diff
@@ -224,4 +228,31 @@ impl ContactsServiceHandle {
             _ => Err(ContactsServiceError::UnexpectedApiResponse),
         }
     }
+
+    pub async fn get_messages(
+        &mut self,
+        pk: TariAddress,
+        x: u64,
+        y: u64,
```

Comment:
Can you add docs here, and or change the x or y variable names? It's unclear what they do. I am assuming it's paging?